utils/limits.tsx

```diff
@@ -16,3 +16,21 @@ export function inK(num: number): string {
 export function toUsagePercent(usage: number, limit: number): number {
     return Math.floor((usage / limit) * 100);
 }
+
+// These are to be used when we need values
+// even if network requests are failing for some reason.
+// Use as a fallback.
+export const fallbackStarterLimits = {
+    messages: {
+        history: 10000,
+    },
+    files: {
+        totalStorage: FileSizes.Gigabyte * 10,
+    },
+    integrations: {
+        enabled: 10,
+    },
+    boards: {
+        cards: 500,
+    },
```

Comment:
Should the fallback be 0? For example 10 integrations would be over the limit and it can show this erroneously in case of network glitch or something

Reply:
Since the variable name says `fallbackStarterLimits` I'm ok leaving as is. This is currently only used for the pricing modal copy.

For a real fallback, I think we should do something server side, possibly in enterprise repo. I think that fallback should default to being more open (like you suggest) to not limit paying customers, rather than deafulting to starter limits for the more common case of non-paying customers.